src/backupNg/backupNg.spec.js

```diff
@@ -169,5 +169,43 @@ describe("backupNg", () => {
       });
       expect(log.warnings).toMatchSnapshot();
     });
+
+    it("fails trying to run a backup job with a VM without disks", async () => {
+      const { id: template } = find(xo.objects.all, { type: "VM-template" });
```

Comment:
What about creating a template dedicated to XO test?

Reply:
After thought, selecting randomly a template can cause troubles. If a template is not good the test will fails randomly.